src/lib/features/list/ui/ListForm/ListForm.tsx

```diff
@@ -0,0 +1,45 @@
+import { useState, type FC } from "react";
+import Input from "../../../../shared/ui/Input/Input";
+import Button from "../../../../shared/ui/Button/Button";
+import * as styles from "./ListForm.css";
+import type { IList } from "../../../../shared/types/lists.type";
+import { listApi } from "../../../../shared/api/lists";
+import { useAppDispatch } from "../../../../shared/store";
+import { listsActions } from "../../../../shared/store/slices/listsSlice";
+
+interface ListFormProps {
+  list?: IList;
+  callback?: (list: IList) => void;
+  boardId: number;
+}
+
+const ListForm: FC<ListFormProps> = ({ list, callback, boardId }) => {
+  const dispatch = useAppDispatch();
+  const [value, setValue] = useState(list?.title || "");
+
+  return (
+    <div className={styles.form}>
+      <Input value={value} onChange={(e) => setValue(e.target.value)} />
+      <Button
+        onClick={() => {
```

Comment:
This style is used everywhere but I'm not a fan of it. I'd rather prefer 2 separate buttons that do separate things